src/main/java/fun/epak/pak/model/user/User.java

```diff
@@ -39,4 +39,13 @@ public class User {
     @OneToMany(cascade = CascadeType.REMOVE, mappedBy = "user")
     @LazyCollection(LazyCollectionOption.FALSE)
     private List<Comment> comments;
+
+    @Transient
+    public String getImagePath(){
```

Comment:
To nie jest raczej miejsce na to. Image path powinien być budowany w jakieś innej klasie i metoda go budująca powinna przyjąć image name od usera.